### PR TITLE
Localization loader and dialog localization fix

### PR DESCRIPTION
**Pull request by rachana-egov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/language_selection.dart [hunk 1]
```diff
@@ -36,56 +45,80 @@ class LanguageSelectionPage extends StatelessWidget {
...
+                    } else if (!state.loading && isDialogVisible) {
+                      isDialogVisible = false;
+                      DigitComponentsUtils().hideLocationDialog(context);
+                    }
+                  },
                   builder: (context, localizationState) {
+                    if (localizationState.loading && !isDialogVisible) {
+                      WidgetsBinding.instance.addPostFrameCallback((_) {
+                        isDialogVisible = true;
+                        DigitComponentsUtils().showLocationCapturingDialog(
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/language_selection.dart:66]: Replace this with a common dialog instead of using locationCapture

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comment

File: apps/health_campaign_field_worker_app/lib/pages/language_selection.dart (hunk 1 above)
Comment (rachana-egov) [reply apps/health_campaign_field_worker_app/lib/pages/language_selection.dart:65]: created a util function to show a localization loader

**Comment by rachana-egov** [comment]
some refresh issue is coming as latest changes are not showing so closing this one and creating one more

**Pull request closed without merging**